docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -2573,7 +2573,7 @@ DATE_PART('year', date'12/05/2006')
 DATE_PART('day', TIMESTAMP '2006-12-31 11:59:59.999999')
 ```
 
-* This function returns the value 201 07.
+* This function returns the value 2011 07.
```

Comment:
Just tried this. Looks like it returns 201107 (no space between the 2011 and the 07).

Reply:
Thanks for your reminding, yes it returns 201107. 
I've corrected. :)